storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc

```diff
@@ -118,21 +118,22 @@ include::modules/lvms-scaling-storage-expand-pvc.adoc[leveloffset=+2]
 include::modules/lvms-upgrading-lvms-on-sno.adoc[leveloffset=+1]
 
 //Volume snapshots
-include::modules/lvms-volume-snapshots-in-single-node-openshift.adoc[leveloffset=+1]
+include::modules/lvms-about-volume-snapshots.adoc[leveloffset=+1]
 
 [role="_additional-resources"]
 .Additional resources
 
 * xref:../../../backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc#oadp-features_oadp-features-plugins[OADP features]
 
-include::modules/lvms-creating-volume-snapshots-in-single-node-openshift.adoc[leveloffset=+2]
-include::modules/lvms-restoring-volume-snapshots-in-single-node-openshift.adoc[leveloffset=+2]
-include::modules/lvms-deleting-volume-snapshots-in-single-node-openshift.adoc[leveloffset=+2]
+include::modules/lvms-creating-volume-snapshots.adoc[leveloffset=+2]
+include::modules/lvms-restoring-volume-snapshots.adoc[leveloffset=+2]
+include::modules/lvms-deleting-volume-snapshots.adoc[leveloffset=+2]
 
 //Volume cloning
-include::modules/lvms-volume-clones-in-single-node-openshift.adoc[leveloffset=+1]
-include::modules/lvms-creating-volume-clones-in-single-node-openshift.adoc[leveloffset=+2]
```

Comment:
FYI, there is one existing external `link` in the openshift-docs repo to this file on the Customer Portal, from a Microshift assembly:

https://github.com/openshift/openshift-docs/blob/main/microshift_storage/volume-snapshots-microshift.adoc?plain=1#L96

It won't 404 (it just dumps to the top of the page on the Portal), but you might want to update that `link:` to the new module location / URL. Possibly cross-check with a member of the Microshift docs team.

Reply:
This is a great catch! Thanks @adellape 
I've informed a writer from the MicroShift team. This link will be updated upon merging this PR.
Thanks!